src/main/antlr/com/sleekbyte/tailor/antlr/Swift.g4

```diff
@@ -679,11 +689,16 @@ functionCallExpression
 
 //initializer_expression : postfix_expression '.' 'init' ;
 
-/*explicitMemberExpression
+/*
+explicitMemberExpression
   : postfixExpression '.' DecimalLiteral // TODO: don't allow '_' here in DecimalLiteral
   | postfixExpression '.' identifier genericArgumentClause?
+  | postfixExpression '.' identifier '(' argumentNames ')'
   ;
-  */
+
+argumentNames : argumentName argumentNames?  ;
```

Comment:
I don't think we need argumentNames and argumentName's definition in the comment.